lib/web_ui/lib/ui.dart

```diff
@@ -14,6 +14,7 @@ import 'dart:math' as math;
 import 'dart:typed_data';
 
 import 'src/engine.dart' as engine;
+import 'ui_web/src/ui_web.dart' as ui_web;
```

Comment:
How are users of `ui_web` going to import the package from outside of `flutter/engine`?

I think we should have a bunch of libraries ready for them to import, like:

* `import package:ui_web/platform_views.dart`
* `import package:ui_web/assets.dart`
* `import package:ui_web/initialization.dart`

(Importing from `src` from outside a package is [discouraged](https://dart.dev/guides/libraries/create-library-packages#organizing-a-library-package).)

Reply:
#### 1. Outside `flutter/engine`:

They simply import `dart:ui_web` like we did here: https://github.com/flutter/flutter/blob/df789c9e76098e82f80f2c5de8b5560f360afa40/packages/flutter_web_plugins/lib/src/navigation/url_strategy.dart#L8

#### 2. Inside `flutter/engine`:

Everything is within the same package (`flutter/lib/web_ui`). The new `ui_web/` is just another subfolder, nothing special about it. Same as `src/engine.dart` right above it.

Reply:
Ahhh import from a **dart:** thing,, so we can't have a nicer file separation in there, shame!